dashboards/MongoDB/MongoDB_Cluster_Summary.json

```diff
@@ -433,58 +420,23 @@
         },
         {
           "expr": "mongodb_mongos_db_data_size_bytes{cluster=\"$cluster\",db!~\"admin|config\"}",
+          "editorMode": "code",
+          "range": false,
+          "exemplar": false,
+          "format": "table",
           "hide": true,
           "instant": true,
-          "interval": "$intgerval",
-          "legendFormat": "{{db}} | {{shard}} | Size",
+          "interval": "$interval",
+          "legendFormat": "{{db}} | {{shard}} | Collections",
```

Comment:
This PromQL doesn't seem to get collections, the one right above does. Could you please double check?

Reply:
Fixed. Thanks